envision/client.py

```diff
@@ -211,16 +201,18 @@ def run_socket(endpoint, num_retries, wait_between_retries, threadlocal):
                 )
 
                 if not connection_established:
-                    self._log.info(
-                        f"Attempting to connect to Envision tries={tries}/{num_retries}"
-                    )
+                    self._log.info(f"Attempting to connect to Envision tries={tries}")
+                else:
+                    # when connection closed, retry again every 5 seconds
+                    wait_between_retries = 5
+                    tries = 0
```

Comment:
Here, I think don't need to reset `tries=0` since number of `tries` is not being checked in the `while` loop.

There might be conflict between #567 and this pull request. I suggest to merge #567 and #545 together first, before merging into the `develop` branch.

Reply:
The `wait_between_retries` is from `_connect`, the default there is 0.5 seconds, which is pretty fast. This is good for trying to establish the initial connection. But I guess when connection is lost, we don't have to try reconnect that frequently. So I set it to 5s.